libmariadb/ma_errmsg.c

```diff
@@ -56,20 +56,20 @@ const char *client_errors[]=
 #elif defined PORTUGUESE
```

Comment:
please remove entire #ifdef PORTUGUESE  block - it is a relict from libmysql 3.23.48 and not in use anymore